### DISCUSSION TITLE
SvelteKit and Adapter-Static; Fetch caching

### DISCUSSION THREAD

**Discussion by jonshipman** [sveltejs/kit#9666] [Q&A]
Is there a builtin way to cache identical requests (internally) during the build process? I noticed that requests in the layout will repeat for every page (like the navigation or blog links in the footer) during build.

Curious if there's something like that already in there that needs a bool true before I add it ontop of my fetch wrapper.

**Comment by geoffrich** [comment]
Not that I can recall. Whatever mechanism you implement for this can likely use the [`building`](https://kit.svelte.dev/docs/modules#$app-environment-building) variable so it only runs during build.
